src/ContentHubClient.php

```diff
@@ -955,127 +955,106 @@ public function __call($method, $args)
    */
     protected function getExceptionMessage($method, array $args, \Exception $exception)
     {
+        $response = NULL;
+        $status_code = 500;
```

Comment:
Could you please use constants defined in Response class instead of 500?

Reply:
@looking-4-justice Don't see constants in Response class

Reply:
@looking-4-justice Nevermind... found in Symfony component, not a Guzzle class